models/snowplow_fractribution_paths_to_conversion.sql

```diff
@@ -0,0 +1,53 @@
+{{
+  config(
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt'))
+  )
+}}
+
+-- Requires macro trim_long_path
+
+with string_aggs as (
+
+  select distinct
+    c.customer_id,
```

Comment:
This is the tiniest performance difference if at all, but could you wrap the `distinct` in a databricks'/spark if block?

Reply:
(Same in the non_conversions)